packages/flutter/test/material/dropdown_test.dart

```diff
@@ -2223,7 +2223,7 @@ void main() {
     final Element element = tester.element(find.byKey(const ValueKey<String>('two')).last);
     final FocusNode node = Focus.of(element);
     expect(node.hasFocus, isTrue);
-  }, skip: kIsWeb);
+  }, skip: isBrowser); // Reference needed
```

Comment:
@yjbanov I have been trying to figure why the three tests noted in this file break on web, and if there is an existing issue. The tests are checking focus, dropdown menus and scrolling. Here are the results of a test run from earlier: https://cirrus-ci.com/task/6419834950909952 I can't tell why they are failing, do you know someone who might?

Reply:
Also, if it helps, this is the related changed that added the skipped tests: https://github.com/flutter/flutter/pull/43722